### PR TITLE
Work in Progress: Header Style

### PR DESCRIPTION
**Pull request by nikolaklenje**
Text_color,background,z-index

### CODE REVIEW

File: components/Hero/index.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { Title } from "../Title";
 
 export const Hero: FC = () => {
 	const { user } = useUser();
-	const [facts, setFacts] = useState<any[]>([]);
+	const [facts, setFacts] = useState<[]>([]);
```
Comment (slavo3dev) [inline components/Hero/index.tsx:13]: this will be array of which elements

File: components/Layout/Logo/logo.module.css [hunk 2]
```diff
@@ -1,19 +1,20 @@
 .logo {
-  font-size: var(--size-5);
+  font-size: var(--size-6) !important;
```
Comment (slavo3dev) [inline components/Layout/Logo/logo.module.css:2]: why forcing important here

File: components/Layout/MainNavigation/burger.tsx [hunk 3]
```diff
@@ -0,0 +1,159 @@
+import Image from "next/image";
```
Comment (slavo3dev) [inline components/Layout/MainNavigation/burger.tsx:1]: rename to MobileView

File: components/Layout/MainNavigation/burger.tsx [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+				</svg>
+			</button>
+
+			<div className={showDrop ? "visible " : "hidden"}>
+				<div className="fixed flex-col h-5/6 overflow-scroll bg-white left-0 z-40 mt-2 w-full rounded border-[.5px] border-light px-5 py-5 transition-all">
+					<ul className="overflow-y-scroll  mr-auto w-full flex list-none flex-col ">
+						<li className="w-11/12 text-sm text-gray-500 rounded-xl">
+							<span
+								onClick={handleMenu}
+								className="relative block hover:text-blue-500 hover:bg-blue-50 w-full p-4 rounded-xl"
```
Comment (slavo3dev) [inline components/Layout/MainNavigation/burger.tsx:45]: please fix code style 
use eslint and prettier in the file

File: components/Layout/MainNavigation/burger.tsx [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+                  Blog 2
+								</li>
+								<li className="w-11/12 p-4 text-sm text-gray-500 hover:text-blue-500 hover:bg-blue-50 rounded-xl">
+                  Blog 3
+								</li>
+								<li className="w-11/12 p-4 text-sm text-gray-500 hover:text-blue-500 hover:bg-blue-50 rounded-xl">
+                  Blog 4
+								</li>
+							</ul>
+						</li>
```
Comment (slavo3dev) [inline components/Layout/MainNavigation/burger.tsx:129]: so many white space

File: components/Layout/MainNavigation/index.tsx [hunk 6]
```diff
@@ -1,43 +1,70 @@
...
+									<span className={classes.user_email}>
+										{user.email}
+									</span>
+								) : (
+									<Link href="/api/auth/login">
+                    Verify Email & Login
+									</Link>
+								)
+							) : (
+								<Link href="/api/auth/login">Login</Link>
```
Comment (slavo3dev) [inline components/Layout/MainNavigation/index.tsx:61]: fix code style

File: components/Layout/MainNavigation/index.tsx [hunk 7]
```diff
@@ -1,43 +1,70 @@
...
+							)}
+						</li>
+					</ul>
+				</nav>
+			</div>
+			<Burger />
```
Comment (slavo3dev) [inline components/Layout/MainNavigation/index.tsx:67]: Mobile View

**Review by slavo3dev — APPROVED** [review]

**Review by slavo3dev — CHANGES_REQUESTED** [review]
please fix code style

File: components/Hero/index.tsx (hunk 1 above)
Comment (nikolaklenje) [reply components/Hero/index.tsx:13]: Previous code

**Review by slavo3dev — APPROVED** [review]

**Pull request merged**